### PR TITLE
feat: Add edit listing modal

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR adds the basics to create the rental editing modal. It loads a rental into the form and, changes the titles and adds two new buttons, one to edit it and the other one to cancel it.
Screen Shot 2022-09-16 at 13 31 10

### CODE REVIEW

File: webapp/src/components/CreateRentalModal/CreateOrEditListingStep/CreateOrEditListingStep.tsx [hunk 1]
```diff
@@ -48,12 +79,18 @@ const CreateListingStep = (props: Props) => {
     }
   }
 
-  // validation
+  // Validations
   const isInvalidPrice = parseMANANumber(pricePerDayInput) <= 0
```
Comment (fzavalia) [inline webapp/src/components/CreateRentalModal/CreateOrEditListingStep/CreateOrEditListingStep.tsx:83]: FYI, the contract accepts a price per day of 0.

File: webapp/src/components/ManageAssetPage/Rent/Rent.tsx [hunk 2]
```diff
@@ -51,7 +51,9 @@ const LinkedProfile = ({ address }: { address: string }) => {
 export const Rent = (props: Props) => {
   const { className, isClaimingLandBack, onClaimLand, rental, nft } = props
 
-  const handleOnEdit = useCallback(() => undefined, [])
+  const handleOnEdit = useCallback(() => {
```
Comment (fzavalia) [inline webapp/src/components/ManageAssetPage/Rent/Rent.tsx:54]: You can remove the block to have a one liner

**Review by fzavalia — APPROVED** [review]
🦒

File: webapp/src/components/CreateRentalModal/CreateOrEditListingStep/CreateOrEditListingStep.tsx (hunk 1 above)
Comment (LautaroPetaccio) [reply webapp/src/components/CreateRentalModal/CreateOrEditListingStep/CreateOrEditListingStep.tsx:83]: Great catch! I'll change it to `<`.

File: webapp/src/components/ManageAssetPage/Rent/Rent.tsx (hunk 2 above)
Comment (LautaroPetaccio) [reply webapp/src/components/ManageAssetPage/Rent/Rent.tsx:54]: Changed it!

**Pull request merged**